connector-packager/tests/test_resources/oauth_connector/oauth-config.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pluginOAuthConfig>
+    <dbclass>databricks2</dbclass>
```

Comment:
```suggestion
    <dbclass>test_oauth</dbclass>
```

Reply:
Caught this, but realized that this connector was still validated even with the mistake. Had to add a bit of code to make sure we mark the connector as invalid when this element does not match the class of the rest of the connector, plus a test case to make sure it isn't broken in the future.

Reply:
@wangxinlei can confirm, but I believe this also enforced at runtime as well.  No reason not to do it here as well though, thanks.

Reply:
Nice catch, currently it is not validated against the dbClass in manifest. When we register the plugin we already got the dbClass from manifest so I can add logic to verify this at runtime. The only reason this field is not removed is because for non-connector plugin we would only have this single oauth-config file.